liwords-ui/src/chat/chat.tsx

```diff
@@ -329,10 +329,10 @@ export const Chat = React.memo((props: Props) => {
   }, [defaultChannel, defaultDescription, loggedIn, props.suppressDefault]);
 
   useEffect(() => {
-    if (props.tournamentID) {
+    if (props.tournamentID || channelType === 'game') {
```

Comment:
let's add `gametv` as a possible channel type for this too

Reply:
done, maybe it's a good idea to create a channelType enum if we start using it in other places